tensorboard/plugins/interactive_inference/witwidget/notebook/jupyter/js/package.json

```diff
@@ -27,8 +27,8 @@
     "webpack": "^3.5.5"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^1.0.0",
-    "@jupyter-widgets/jupyterlab-manager": "^0.38.1"
+    "@jupyter-widgets/base": ">=1.0.0",
```

Comment:
The line before your change should roughly achieve the same effect. Is there a reason why you prefer this? 
i.e., ^1.0.0 would match any patch & minor release like 1.9.1. What you have here would make 15.3.1 also match but that is generally unfavorable since major bump often come with breaking changes and you want to make sure you are not impacted.

Reply:
What would be the right change for jupyterlab-manager in that case? I want it to work with 0.38 but also 1.x.

Reply:
I suppose, if that's what you want, you can do `>=0.38.1 <2.0.0`.

Reply:
thanks, done